inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -243,7 +243,7 @@ persistent_volumes_enabled: false
 
 # A comma separated list of levels of node allocatable enforcement to be enforced by kubelet.
 # Acceptable options are 'pods', 'system-reserved', 'kube-reserved' and ''. Default is "".
-# kubelet_enforce_node_allocatable: pods
+kubelet_enforce_node_allocatable: pods,system-reserved,kube-reserved
```

Comment:
Enforcement will actually kill system and kube daemons that run beyond their reservation. I don't think we'll ever have sensible defaults for that unless we set them dynamically. Even then it's going to vary on the environment

Reply:
I don't prefer this to be set as the default.